db/controllers/articles.controller.js

```diff
@@ -0,0 +1,23 @@
+const { error } = require('console')
+const { retrieveArticlesById } = require('../models/articles.model')
+const fs = require('fs/promises')
```

Comment:
You don't need to use fs/promises :) I would take this out as it could be misleading for a senior dev to read :)